terminus-ui/icon/src/custom-icons/logo.ts

```diff
@@ -1,6 +1,6 @@
 // tslint:disable: max-line-length
 export const LOGO_ICON = `
-    <svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
-        <path d="M7.95132809,10.2081825 L11.5080084,8.95561536 C11.6578126,8.89933366 11.8169796,9.01309453 11.8169796,9.17714969 L11.8169796,19.7628984 C11.8169796,19.893424 11.7139892,19.9988025 11.5864216,19.9988025 L8.03091159,19.9988025 C7.90334392,19.9988025 7.80035351,19.893424 7.80035351,19.7628984 L7.80035351,10.4297168 C7.80035351,10.3315232 7.86121148,10.2429095 7.95132809,10.2081825 Z M0,13.1647677 L0,19.7640959 C0,19.8946215 0.102990414,20 0.230558086,20 L3.75329838,20 C3.88086605,20 3.98385646,19.8946215 3.98385646,19.7640959 L3.98385646,11.9469276 C3.98385646,11.7876624 3.83171153,11.6739015 3.68307764,11.7229983 L0.160337349,12.9420359 C0.0643690087,12.971973 0,13.0617842 0,13.1647677 Z M11.6905823,0.288234239 L11.9913611,1.7743105 L0.146293202,5.96789556 C0.057346935,6.00262256 0,6.09003881 0,6.18703493 L0,9.34120485 C0,9.50406252 0.156826312,9.61782339 0.30663055,9.56273918 L12.8223065,5.1464223 L13.0938267,6.36066995 C13.1347888,6.54508231 13.3606655,6.60735142 13.4870628,6.46844341 L17.9378872,1.56954092 C18.0607735,1.43422536 17.9917231,1.21508599 17.8150009,1.17796402 L11.9609322,0.00443079495 C11.7970838,-0.0279012429 11.6543016,0.121784118 11.6905823,0.288234239 Z" transform="translate(3 2)"/>
-    </svg>
+  <svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Since we are building a separate module, we should move the SVGs into this module. `terminus-ui/logo/src/svgs/my-logo.svg` That way they aren't bundled with the icon module

Reply:
I actually see now that there are already logos in the logo module.. are the icon changes related to the logo module?

Reply:
The icon changes were updated svg's from UX